src/molecules/tables/tables.stories.tsx

```diff
@@ -125,6 +126,11 @@ export const autoColumnData: AutoColumnGenerator = {
   tableType: jsonSchema,
   excludeList: ['id'],
   selectable: false,
+  customCells: {
+    status: ({ row }) => {;
+      return <Link href={"https://google.com"} className='text-blue-600'>{row.getValue('status')}</Link>;
+    },
+  }
 };
```

Comment:
we can use this definition for any custom cells (links, icons, etc) based on the need of the project